sql_generators/baseline_clients_city_seen_v1/__init__.py

```diff
@@ -0,0 +1,154 @@
+"""Generate baseline clients city seen per app."""
+
+import os
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, List
+
+import click
+from jinja2 import Environment, FileSystemLoader
+
+from bigquery_etl.cli.utils import use_cloud_function_option
+from bigquery_etl.config import ConfigLoader
+from bigquery_etl.format_sql.formatter import reformat
+from bigquery_etl.util.common import render, write_sql
+from sql_generators.glean_usage import get_app_info
+
+THIS_PATH = Path(os.path.dirname(__file__))
+TABLE_NAME = os.path.basename(os.path.normpath(THIS_PATH))
```

Comment:
nit: It seems redundant to call normpath and then basename.  Also `Path` and `os.path` don't need to be mixed here

```suggestion
THIS_PATH = Path(__file__).parent
TABLE_NAME = THIS_PATH.name
```